static/templates/items/item-sheet.hbs

```diff
@@ -82,11 +82,37 @@
         <section class="sheet-body">
             <!-- Item Description -->
             <div class="tab item-description" data-tab="description">
-                {{#if (not isVariant)}}
-                    {{editor enrichedContent.description target="system.description.value" button=true owner=owner editable=editable}}
-                {{else}}
-                    {{{enrichedContent.description}}}
-                {{/if}}
+                <div class="item-description-layout">
+                    {{#if user.isGM}}
+                        <div class="section">
```

Comment:
A `div` with a class of "section" is rather perverse. :)

Reply:
Haha, good point! 😂 I'll change it to use `<section>` instead.